tests/python/pants_test/pants_run_integration_test.py

```diff
@@ -166,7 +166,15 @@ def _read_log(filename):
 class PantsRunIntegrationTest(unittest.TestCase):
   """A base class useful for integration tests for targets in the same repo."""
 
-  PANTS_SCRIPT_NAME = 'pants'
+  # NB: We must depend on `pants.pex`, rather than `pants`, because we strip the prefix of source
```

Comment:
Links to related issues and discussions here could be helpful for future me (in particular, this one: https://github.com/pantsbuild/pants/issues/7281)